components/icon-buttons/Web/Web-icon-buttons.md

```diff
@@ -8,227 +8,167 @@ path: /components/buttons
+```
 
-You can optionally add a JavaScript ripple effect (see [MDC Ripple](https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple)) to your buttons by importing and then instantiating `MDCRipple` in your `*.js` file. See the page on importing the [JavaScript component](https://github.com/material-components/material-components-web/blob/master/docs/importing-js.md) for more information.
+### JavaScript Instantiation
 
-To bundle your `*.js` file, go to the [quickstart page](https://github.com/material-components/material-components-web/blob/master/docs/getting-started.md#quick-start-cdn).
+The icon button will work without JavaScript, but you can enhance it to have a ripple effect by instantiating `MDCRipple` on the root element.
+See [MDC Ripple](../mdc-ripple) for details.
 
 ```js
 import {MDCRipple} from '@material/ripple';
 
-const iconButtonRipple = new MDCRipple(document.querySelector(`.mdc-icon-button`));
+const iconButtonRipple = new MDCRipple(document.querySelector('.mdc-icon-button'));
 iconButtonRipple.unbounded = true;
 ```
 
-### Making icon buttons accessible
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
-Material Design spec advises that touch targets should be at least 48 x 48 px.
-To meet this requirement, add the following to your button:
+### Icons
+
+We recommend using [Material Icons](https://material.io/tools/icons/) from Google Fonts:
 
 ```html
-<div class="mdc-touch-target-wrapper">
-  <button class="mdc-icon-button mdc-icon-button--touch">
-    <div class="mdc-icob-button__ripple"></div>
-    <span class="mdc-icon-button__label">My Accessible Icon Button</span>
-    <div class="mdc-icon-button__touch"></div>
-  </button>
-</div>
+<head>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
 ```
 
-_**Note**: The outer `mdc-touch-target-wrapper` element is only necessary if you want to avoid potentially overlapping touch targets on adjacent elements (due to collapsing margins)._
-
-
-## Toggle button
-
-
-[Toggle buttons](https://material.io/components/buttons/#toggle-button) can be used to select from a group of choices.
-
-There are two types of toggle buttons:
+However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any other icon library you wish.
 
-* [Toggle button](#toggle-button)
-* [Icon](#icon)
+## Icon Button
 
-### Toggle button
-
-The Material.io framework for Web currently does not support toggle button groups, such as:
-
-<img src="toggle-bar.png" alt="toggle button group">
-
-If your application needs a toggle button group, you will need to use a different framework such as [Vuetify](https://vuetifyjs.com/en/components/button-groups#button-groups).
+```html
+<button class="mdc-icon-button material-icons">favorite</button>
+```
 
-### Icon
+> Note: The MDC Icon Button can be used with both `<button>` and `<a>` tags.
```

Comment:
Can we avoid using '>'? It renders the content as quotes. Please use bold (**) instead.